contrib/bitcoin-mock.md

```diff
@@ -53,17 +53,37 @@ docker exec -it bitcoind-node /bin/bash
 Then you can generate a block:
 
 ```sh
-bitcoin-cli -regtest generate <number-block>
+bitcoin-cli -regtest -generate <number-block>
 ```
 
 If RPC params are required, you can provide them:
 
 ```sh
-bitcoin-cli -regtest -rpcuser=user -rpcpassword=password generate
+bitcoin-cli -regtest -rpcuser=user -rpcpassword=password -generate <number-block>
 ```
 
 More information in [developer.bitcoin.org -> testing](https://developer.bitcoin.org/examples/testing.html).
 
+### Create BTC fork for testing
+
+In a few cases, we must create a BTC fork for testing. The create-fork.sh script helps you to do this. We support two functions:
+
+#### Create fork
+
+Create a new fork start at the latest block in the snapshot data.
```

Comment:
```suggestion
Creates a new fork starting at the latest block in the snapshot data.
```

Reply:
snapshot data? I see we can provide a `number-block`

Reply:
So this is `bitcoind-snapshot` folder. We assume all fork start from the latest block of this hash chain.